src/components/DayPicker.jsx

```diff
@@ -446,7 +446,7 @@ class DayPicker extends BaseClass {
     }
   }
 
-  onPrevMonthClick(nextFocusedDate, e) {
+  onPrevMonthClick(e, nextFocusedDate) {
```

Comment:
it's a pretty strong convention to have `e` be last; why is this breaking change needed?

Reply:
The motivation for this was because we were creating a new function in every render and causing unnecessary updates as a result. Flipping the order of arguments was a pretty simple solution, and was motivated by the fact that both arguments are occasionally omitted. 

We could alternatively cache the function with the event as the second arg as well if you think it makes more sense @ljharb. It may make sense to have `onNextMonthClick` take the event as an argument and call a function like `transitionToNextMonth` that can also be called directly. 🤔 Actually, that may be less confusing in the long run